agent/engine/docker_task_engine.go

```diff
@@ -945,6 +961,22 @@ func (engine *DockerTaskEngine) createContainer(task *apitask.Task, container *a
 	return metadata
 }
 
+func getLogRouterConfig(task *apitask.Task, container *apicontainer.Container, hostConfig *dockercontainer.HostConfig, engine *DockerTaskEngine) dockercontainer.LogConfig {
+	seelog.Debugf("Applying logrouter config for container %s", container.Name)
+	fields := strings.Split(task.Arn, "/")
+	taskID := fields[len(fields)-1]
+	tag := container.Name + "-" + taskID
+	fluentd := filepath.Join(engine.cfg.DataDirOnHost, dataLogDriverPath, taskID, dataLogDriverSocketPath)
```

Comment:
this needs to have a prefix "unix://". when the driver uses a socket it needs to have that prefix